Makefile

```diff
@@ -74,30 +74,16 @@ clean:
 generatecode:
 	mkdir -p $(OUTPUT_DIR)
 	go run cmd/generate-code/main.go
-
-# Docs
-.PHONY: generatedocs
-generatedocs:
-	go run cmd/generate-docs/main.go
-	mkdocs build
-
-.PHONY: previewsite
-previewsite:
-	cd site && python3 -m http.server 0
+	# go run cmd/generate-docs/main.go
```

Comment:
Not running this for now b/c it won't work on CI, see this failure: 
https://console.cloud.google.com/cloud-build/builds/2a3c354b-4d49-4f5d-8b14-a8fa0fb0b64f?project=solo-public